css/at-rules/scroll-timeline.json

```
@@ -0,0 +1,203 @@
+{
+  "css": {
+    "at-rules": {
+      "scroll-timeline": {
+        "__compat": {
+          "description": "<code>@scroll-timeline</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@scroll-timeline",
+          "spec_url": "https://drafts.csswg.org/scroll-animations/#scroll-timeline-at-rule",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "97"
+            },
+            "firefox_android": {
+              "version_added": "97"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
```

These are all single-implementation right now, so let's set `experimental` to true. See the [_Choosing an experimental status_ data guideline](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status) for details.

```suggestion
            "experimental": true,
```